BeeTracker/app/src/main/res/layout/review_fragment.xml

```diff
@@ -1,19 +1,114 @@
+    android:text="@string/submit"
+    android:gravity="center_vertical"
+    android:layout_alignParentRight="true"
+    android:layout_marginTop="8dp"
+    android:layout_marginBottom="8dp"
+    android:layout_marginRight="8dp"
+    android:id="@+id/submitButton"/>
+</RelativeLayout>
+    <LinearLayout
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Review fragment"/>
-    <Button
-        android:layout_width="match_parent"
+        android:orientation="horizontal"
+        android:layout_marginTop="30sp"
+        android:layout_marginLeft="20sp">
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@mipmap/ic_bumblebee"/>
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="IDENTIFIED SPECIES"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textStyle="bold|italic"
+                android:text="@string/speciesID"
+                android:id="@+id/speciesID"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/scientificID"
+                android:id="@+id/scientificID"/>
+        </LinearLayout>
+    </LinearLayout>
+    <LinearLayout
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/submit"
-        android:id="@+id/submitButton"/>
-    <Button
+        android:orientation="vertical"
+        android:layout_marginTop="30sp">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="20sp"
+            android:text="location"/>
+        <LinearLayout
+            android:layout_width="394dp"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:background="#FFFFFF">
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="20sp"
+                android:src="@mipmap/ic_location_arrow"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textStyle="bold"
+                android:layout_gravity="center_vertical"
+                android:text="GPS co-ords"/>
+        </LinearLayout>
+    </LinearLayout>
+    <LinearLayout
```

Comment:
Same comment about this LinearLayout, you can just put the textview and spinner directly in the parent layout